bin/xcode-build-phase/increment-build-number.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+
+HEADER="${PROJECT_DIR}/${INFOPLIST_PREFIX_HEADER}"
```

Comment:
Two ways:

### Copy and Paste

Add an empty Run Script build phase or choose an existing Run Script build phase.

Turn on "Show environment in build log"

In the Build Logs, copy all messages and paste to a text editor. 

Search.

### Build Settings + Quick Help in Xcode

![image](https://cloud.githubusercontent.com/assets/466104/15170426/ee242dd0-1745-11e6-8757-6ba92f215b70.png)